x-pack/legacy/plugins/siem/package.json

```diff
@@ -6,20 +6,37 @@
   "license": "Elastic-License",
   "scripts": {
     "build-graphql-types": "node scripts/generate_types_from_graphql.js",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run --reporter mocha-multi-reporters --reporter-options configFile=./reporter_config.json; mochawesome-merge --reportDir ../../../../target/kibana-siem/cypress/results > ../../../../target/kibana-siem/cypress/results/output.json; marge ../../../../target/kibana-siem/cypress/results/output.json --reportDir ../../../../target/kibana-siem/cypress/results; mkdir -p ../../../../target/junit && cp ../../../../target/kibana-siem/cypress/results/*.xml ../../../../target/junit/"
   },
   "devDependencies": {
     "@cypress/webpack-preprocessor": "^4.1.0",
     "@types/js-yaml": "^3.12.1",
     "@types/lodash": "^4.14.110",
     "@types/react-beautiful-dnd": "^10.0.1",
-    "cypress": "^3.3.1",
+    "cypress": "^3.4.1",
     "js-yaml": "^3.13.1",
+    "mocha-junit-reporter": "^1.23.1",
+    "mocha-multi-reporters": "^1.1.7",
+    "mochawesome": "^4.0.1",
+    "mochawesome-merge": "^2.0.1",
+    "mochawesome-report-generator": "^4.0.1",
     "ts-loader": "^6.0.4"
   },
   "dependencies": {
     "lodash": "^4.17.13",
     "react-beautiful-dnd": "^10.0.1",
     "react-markdown": "^4.0.6"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
```

Comment:
@spalger as noted in our out-of-band discussion, we now use `nohoist` here in `siem/package.json` so that the (new) local `devDependencies` named in `nohoist` are installed in `siem/node_modules` instead of the root `node_modules`.

This configuration is required for `cypress` to run with multiple reporters, and to run  a reporter that's not included in the base `cypress` install.

All of the modules noted in `nohoist` are `devDependencies`, so they shouldn't get slurped up by `webpack`, but I wanted to call-out the use of `nohoist`, since it's not typically required.